adapters/impactify/impactify.go

```diff
@@ -0,0 +1,186 @@
+package impactify
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strings"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type ImpactifyExtBidder struct {
+	Impactify openrtb_ext.ExtImpImpactify `json:"impactify"`
+}
+
+type DefaultExtBidder struct {
+	Bidder openrtb_ext.ExtImpImpactify `json:"bidder"`
+}
+
+func (a *adapter) MakeRequests(bidRequest *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+	var validImps []openrtb2.Imp
+
+	var adapterRequests []*adapters.RequestData
+
+	for i := 0; i < len(bidRequest.Imp); i++ {
+		// Check if imp comes with bid floor amount defined in a foreign currency
+		if bidRequest.Imp[i].BidFloor > 0 && bidRequest.Imp[i].BidFloorCur != "" && strings.ToUpper(bidRequest.Imp[i].BidFloorCur) != "USD" {
```

Comment:
Please add a json test for currency conversion.

Reply:
I don't really know how to make this test, because i can not catch the "bidfloor" who will be sent. This value depends on the current rates.

Reply:
I see. Looks like we need to build out a bit more of the testing framework for this case. I'll give this one a pass.